src/de.rs

```diff
@@ -62,25 +62,25 @@ impl<'de> de::Deserializer<'de> for Value {
     #[inline]
     fn deserialize_u8<V: de::Visitor<'de>>(self, visitor: V) -> Result<V::Value> {
         // FIXME: This should *fail* if the value does not fit in the requets integer type
```

Comment:
Now's not a bad time to take of these FIXME's by replacing these `as` casts with `try_into`. I could roll that into this PR if desired.